src/msg/producer/writer/message_writer.go

```diff
@@ -381,7 +377,10 @@ func (w *messageWriterImpl) Init() {
 func (w *messageWriterImpl) scanMessageQueueUntilClose() {
 	var (
 		interval = w.opts.MessageQueueNewWritesScanInterval()
-		jitter   = time.Duration(w.r.Int63n(int64(interval)))
+		jitter   = time.Duration(
```

Comment:
Nit: around 49 days actually? Assuming interval.Milliseconds <= 4294967295 which comes out to 49 [days](https://www.google.com/search?q=4294967295+milliseconds&rlz=1C5GCEM_enUS927US927&oq=4294967295+milliseconds&aqs=chrome..69i57.2035j0j7&sourceid=chrome&ie=UTF-8). Mostly just checking for my understanding here; not actually important :D

Reply:
I just googled for 4 billion to milliseconds. Was close enough for me :)

Reply:
:D ok cool just wanted to make sure I had the math right.